### PR TITLE
Updates Siderus Orion details

### PR DESCRIPTION
**Pull request by koalalorenzo**
This will update the details about Siderus Orion

### CODE REVIEW

**Comment by koalalorenzo** [comment]
Can somebody re-try the CI part? There was a connection error from Jenkins!

File: data/tools.yaml [hunk 1]
```diff
@@ -68,10 +68,10 @@ content:
 - title: ipfs-linux-service
   source: https://github.com/dylanPowers/ipfs-linux-service
   description: IPFS Linux Init Daemon
-- title: Orion
+- title: Siderus Orion
   picture: /images/orion.png
-  source: https://github.com/Siderus/Orion
-  description: KISS, Easy to setup and use IPFS node for macOS, Windows and Linux.
+  source: https://orion.siderus.io
```
Comment (hacdias) [inline data/tools.yaml]: It should be:

```
website: https://orion.siderus.io
source: https://github.com/Siderus/Orion
```

**Comment by hacdias** [comment]
@koalalorenzo I did restart Jenkins. I think you can too: you just need to Login with your GitHub account. If you have write permissions here, you'll be able to start/restart and stop Jenkins

**Comment by hacdias** [comment]
:O The build is failing, but I believe it's just something on Jenkins. I'd merge this.

**Pull request merged**